lib/config/config.js

```diff
@@ -279,6 +279,37 @@ module.exports = function (fs, path, url, convict) {
       default: '',
       format: 'String',
       env: 'SENTRY_DSN'
+    },
+    customRules: {
```

Comment:
Naming nit: using "custom" in the sense of "customized" may be confusing when mixed with "customs" in the sense of "customs and border security", I wonder if we can find an equivalent term that's less ambiguous.